reference/5.1/PSReadline/About/about_PSReadline.md

```diff
@@ -0,0 +1,959 @@
+appends the previous kill, instead of adding a new item or replacing an item
+in the kill ring. This is how you can cut a part of a line, for example, with
+multiple KillWord operations, then yank them back elsewhere as a single yank.
+
+### KillLine
+
+- Cmd: unbound
+- Emacs: `<Ctrl+K>`
+
+Clear the input from the cursor to the end of the line. The cleared text is
+placed in the kill ring.
+
+### BackwardKillLine
+
+- Cmd: unbound
+- Emacs: `<Ctrl+U>` or `<Ctrl+X,Backspace>`
+
+Clear the input from the start of the input to the cursor. The cleared text is
+placed in the kill ring.
+
+### KillWord
+
+- Cmd: unbound
+- Emacs: `<Alt+D>`
+
+Clear the input from the cursor to the end of the current word. If the cursor
+is between words, the input is cleared from the cursor to the end of the next
+word. The cleared text is placed in the kill ring.
+
+### BackwardKillWord
+
+- Cmd: unbound
+- Emacs: `<Alt+Backspace>`
+
+Clear the input from the start of the current word to the cursor. If the
+cursor is between words, the input is cleared from the start of the previous
+word to the cursor. The cleared text is placed in the kill ring.
+
+### ShellKillWord
+
+- Cmd: unbound
+- Emacs: unbound
+
+Like KillWord, except word boundaries are defined by PowerShell
+token boundaries.
+
+### ShellBackwardKillWord
+
+- Cmd: unbound
+- Emacs: unbound
+
+Like BackwardKillWord, except word boundaries are defined by PowerShell token
+boundaries.
+
+### UnixWordRubout
+
+- Cmd: unbound
+- Emacs: `<Ctrl+W>`
+
+Like BackwardKillWord, except word boundaries are defined by white space.
+
+### KillRegion
+
+- Cmd: unbound
+- Emacs: unbound
+
+Kill the text between the cursor and the mark.
+
+### Copy
+
+- Cmd: `<Ctrl+Shift+C>`
+- Emacs: unbound
+
+Copy selected region to the system clipboard. If no region is selected, copy
+the whole line.
+
+### CopyOrCancelLine
+
+- Cmd: `<Ctrl+C>`
+- Emacs: `<Ctrl+C>`
+
+Either copy selected text to the clipboard, or if no text is selected, cancel
+editing the line with CancelLine.
+
+### Cut
+
+- Cmd: `<Ctrl+X>`
+- Emacs: unbound
+
+Delete selected region placing deleted text in the system clipboard.
+
+### Yank
+
+- Cmd: unbound
+- Emacs: `<Ctrl+Y>`
+
+Add the most-recently killed text to the input.
+
+### YankPop
+
+- Cmd: unbound
+- Emacs: `<Alt+Y>`
+
+If the previous operation was Yank or YankPop, replace the previously-yanked
+text with the next killed text from the kill ring.
+
+### ClearKillRing
+
+- Cmd: unbound
+- Emacs: unbound
+
+The contents of the kill ring are cleared.
+
+### Paste
+
+- Cmd: `<Ctrl+V>`
+- Emacs: unbound
+
+This is similar to Yank, but uses the system clipboard instead of the kill
+ring.
+
+### YankLastArg
+
+- Cmd: `<Alt+.>`
+- Emacs: `<Alt+.>`, `<Alt+_>`
+
+Insert the last argument from the previous command in history. Repeated
+operations replace the last inserted argument with the last argument from the
+previous command (so Alt+. Alt+. will insert the last argument of the second
```

Comment:
Should the cmd still be codefenced even in the description?

Reply:
There would be 100s of these. I think we just leave it as is.